exporters/filters/multiple_filter.py

```diff
@@ -0,0 +1,104 @@
+from exporters.filters.base_filter import BaseFilter
+from exporters.module_loader import ModuleLoader
+from exporters.utils import dict_list
+
+
+class MultipleFilter(BaseFilter):
+    """
+    Apply multiple filters to an Item, the filters can be combined. The way
+    this filter works is very similar to [Mongo Query](
+    https://docs.mongodb.com/manual/tutorial/query-documents)
+
+        - filters (list of dict)
+            Filters to be applied to each item
+
+    Example:
+        "filters": [
+            {"name": "exporters.filters.key_value_regex_filter.KeyValueRegexFilter",
+             "options": {
+                 "keys": [
+                     {"name": "country", "value": "United States"}
+                 ]
+             }
+            },
+            {"name": "exporters.filters.key_value_regex_filter.KeyValueRegexFilter",
+             "options": {
+                 "keys": [
+                     {"name": "status", "value": "Verified"}
+                 ]
+             }
+            },
+            {"or": [
+                {"name": "exporters.filters.key_value_regex_filter.KeyValueRegexFilter",
+                 "options": {
+                     "keys": [
+                         {"name": "city", "value": "New York"}
+                     ]
+                 }
+                },
+                {"name": "exporters.filters.key_value_regex_filter.KeyValueRegexFilter",
+                 "options": {
+                     "keys": [
+                         {"name": "city", "value": "Los Angeles"}
+                     ]
+                 }
+                },
+            ]}
+        ]
+
+        The config above could be translated to:
+        country == 'United States' and status == 'Verified' and
+        (city == 'New York' or city == 'Los Angeles')
+    """
+
+    # List of options
+    supported_options = {
+        'filters': {'type': dict_list},
+    }
+
+    def __init__(self, *args, **kwargs):
+        super(MultipleFilter, self).__init__(*args, **kwargs)
+        self.module_loader = ModuleLoader()
+        self.filter_options = self.read_option('filters')
+        self.filters = self._load_filters(self.filter_options)
+        self.filter_func = self._create_filter_func(self.filters)
+        self.logger.info('MultipleFilter instantiated.')
+
+    def filter(self, item):
+        return self.filter_func(item)
+
+    def _create_filter_func(self, filters):
+        """ Should generate a list of functions and return a function
+        that check if all the functions in list return True """
+        filter_functions = []
+        for f in filters:
+            _filter = f.get('name') or f.get('or')
+            if isinstance(_filter, list):
```

Comment:
If we can know for sure what's the thing that we're getting (`either f.get('name')` or `f.get('or')`), why rely on type-checking a specific type?
This would be better as:
```
if f.get('name'):
    filter_func = ...
elif f.get('or'):
    filter_func = ...
else:
    raise ValueError("....")
```